toqito/matrix_ops/tests/test_tensor.py

```diff
@@ -5,174 +5,64 @@
 from toqito.matrix_ops import tensor
 from toqito.states import basis
 
+e_0, e_1 = basis(2, 0), basis(2, 1)
+matrix1 = np.array([[1, 2]])
```

Comment:
Since these appear to only be used in one place, I wonder if we just instead define these as inline values as done in the rest of the tests. So, for instance:

```py
    (np.array([matrix1, matrix2, matrix3, matrix4], dtype=object), 1, np.kron(
        np.kron(matrix1, np.kron(matrix2, matrix3)), matrix4)),
```

would become:

```py
    (np.array([np.array([[3], [4]]), np.array([[3], [4]]), np.array([[5, 6]]), np.array([[7, 8]])], dtype=object), 1, np.kron(
        np.kron(np.array([[3], [4]]), np.kron(np.array([[3], [4]]), np.array([[5, 6]]))),np.array([[7, 8]]))),
```

Reply:
Yeah, I thought about that but then your suggestion also makes that particular parameter unreadable. Where does the parentheses begin, stop etc.